activity_calendar/feeds.py

```diff
@@ -32,7 +37,14 @@ def func_wrapper(self, item):
 def get_feed_id(item):
     # ID should be _globally_ unique
     if isinstance(item, Activity):
-        return f"local_activity-id-{item.id}@kotkt.nl"
+        # Sometimes a specific feed_name is given along with the object. Prioritise that.
```

Comment:
I'm not entirely convinced of this setup, but I don't really see a good alternative on a quick glance either.